haunted-houses/haunted-houses-service/src/main/java/cz/muni/fi/pa165/hauntedhouses/service/facade/HouseFacadeImpl.java

```diff
@@ -3,32 +3,52 @@
 import cz.muni.fi.pa165.hauntedhouses.dto.HouseCreateDTO;
 import cz.muni.fi.pa165.hauntedhouses.dto.HouseDTO;
 import cz.muni.fi.pa165.hauntedhouses.facade.HouseFacade;
+import cz.muni.fi.pa165.hauntedhouses.model.House;
+import cz.muni.fi.pa165.hauntedhouses.service.HouseService;
+import cz.muni.fi.pa165.hauntedhouses.service.MappingService;
+import org.springframework.beans.factory.annotation.Autowired;
 
 import java.util.List;
 
 public class HouseFacadeImpl implements HouseFacade {
+
+    @Autowired
+    MappingService mappingService;
+
+    @Autowired
+    HouseService houseService;
+
     @Override
     public HouseDTO findHouseById(Long id) {
-        return null;
+        return mappingService.mapTo(houseService.getHouseById(id), HouseDTO.class);
     }
 
     @Override
     public List<HouseDTO> findAllHouses() {
-        return null;
+        return mappingService.mapTo(houseService.getAllHouses(), HouseDTO.class);
     }
 
     @Override
-    public void createHouse(HouseCreateDTO house) {
-
+    public Long createHouse(HouseCreateDTO house) {
+        House newHouse = new House();
```

Comment:
mappingService by měla zvládnout namapovat DTO na entitu

Reply:
Aj v prípade EntityCreateDTO, nie len EntityDTO?

Reply:
Zmenil som to